fillager/pom.xml

```diff
@@ -63,16 +63,12 @@
 			<type>jar</type>
 		</dependency>
 		<dependency>
-			<groupId>io.springfox</groupId>
```

Comment:
https://stackoverflow.com/questions/70036953/springboot-2-6-0-spring-fox-3-failed-to-start-bean-documentationpluginsboot

Det er enkelt workaround for denne buggen som er publisert.

Reply:
Jeg såg det samme. Men det föles akkurat som en workaround og ikke en god fiks. Springfox har ikke fått nye commits siden [Oktober 2020](https://github.com/springfox/springfox), så det virker ikke vare maintained lengre(?). Derfor bytte jeg mot springdoc-openapi-ui. Er du ikke enig i det? :)